GitHubActions/build.sh

```diff
@@ -35,15 +35,13 @@ MODE="$1"
 
 if [ "$MODE" = "tests" -o "$MODE" = "all" ]; then
   echo "Running MessageKit tests."
-  carthage bootstrap --platform ios
-  set -o pipefail && xcodebuild test -project MessageKit.xcodeproj -scheme MessageKitTests -destination "platform=iOS Simulator,name=iPhone 11 Pro" CODE_SIGNING_REQUIRED=NO | xcpretty -c
+  set -o pipefail && xcodebuild test -scheme MessageKit -sdk iphonesimulator -destination "platform=iOS Simulator,name=iPhone 11" | xcpretty -c
```

Comment:
You need to switch to XCode 12 (non-default) for Swift 5.3 - https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md

Reply:
Fixed here 👍 
https://github.com/MessageKit/MessageKit/pull/1479/commits/37648f1dbe88525cc7b631aac95a1224f4a97dbd#